paper/paper.md

```diff
@@ -0,0 +1,137 @@
+As these formats contain internally the representation of R objects, they cannot be directly used from Python, another widely used language for data analysis and processing.
+The library `rdata` allows to load and convert these datasets to Python objects, without the need of exporting them to other intermediate formats which may not keep all the original information.
+This library has minimal dependencies, ensuring that it can be used in contexts where an R installation is not available.
+The capability to write data in Rda and Rds formats is also under development.
+Thus, the library `rdata` facilitates data interchange, enabling the usage of the same datasets in both languages (e.g. for reproducibility, comparisons of results against methods in both languages, or migration of processing pipelines to Python).
+
+# Statement of need
+
+The datasets from the CRAN repository are stored in the R specific format RData.
+In Python, there were a few packages that could parse this file format, albeit all of them presented some limitations.
+
+The package `rpy2` [@gautier_2024_rpy2] can be used to interact with R from Python.
+This includes the ability to load data in the RData format, and to convert these data to equivalent Python objects.
+Although this is arguably the best package to achieve interaction between both languages, it has many disadvantages if one wants to use it just to load RData datasets.
+In the first place, the package requires an R installation, as it relies in launching an R interpreter and communicating with it.
+Secondly, launching R just to load data is inefficient, both in time and memory.
+Finally, this package inherits the GPL license from the R language, which is not compatible with most Python packages, typically released under more permissive licenses.
+
+The package `pyreadr` [@fajardo_2024_pyreadr] also provides functionality to read and write some R datasets.
+It relies in the C library `librdata` in order to perform the parsing of the RData format.
+This adds an additional dependency from C building tools, and requires that the package is compiled for all the desired operating systems.
+Moreover, this package is limited by the functionalities available in `librdata`, which at the moment of writing
+does not include the parsing of common objects such as R lists and S4 objects.
+The license can also be a problem, as it is part of the GPL family and does not allow commercial use.
+
+As existing solutions were unsuitable for our needs, the package `rdata` was developed to parse data in the RData format.
+This is a small, extensible, efficient, and very complete implementation in pure Python of a RData parser, that is able to read and convert most datasets in the CRAN repository to equivalent Python objects, such as the built-in types of The Python Standard Library, NumPy arrays [@harris+_2020_numpy], or Pandas dataframes [@mckinney_2010_pandas; @pandasdevelopmentteam_2024_pandasdev].
```

Comment:
```suggestion
This is a small, extensible, efficient, and very complete implementation in pure Python of a R data parser, that is able to read and convert most datasets in the CRAN repository to equivalent Python objects, such as the built-in types of the Python standard library, NumPy arrays [@harris+_2020_numpy], or Pandas dataframes [@mckinney_2010_pandas; @pandasdevelopmentteam_2024_pandasdev].
```

Reply:
Looks good, but suggesting to change to "an R data parser" because "R" has vowel sound.